iron-collapse.html

```diff
@@ -174,19 +182,23 @@
         // For `auto` we must calculate what is the final size for the animation.
         // After the transition is done, _transitionEnd will set the size back to `auto`.
         if (size === 'auto') {
```

Comment:
Agreed 👍 Would you mind adding a jsdoc for this method, documenting what `size` should be? 🙏 
```javascript
/**
 * Updates the size of the element.
 * @param {!String} size The new value for `maxWidth`/`maxHeight`
 * @param {boolean=} animated
 */